indexer/examples/p-chain/main.go

```diff
@@ -0,0 +1,54 @@
+// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/ava-labs/avalanchego/indexer"
+	"github.com/ava-labs/avalanchego/vms/platformvm/blocks"
+	"github.com/ava-labs/avalanchego/vms/proposervm/block"
+)
+
+// This example program continuously polls for the next P-Chain block
+// and prints the ID of the block and its transactions.
+func main() {
+	var (
+		uri              = fmt.Sprintf("%s/ext/index/P/block", "https://indexer-demo.avax.network")
```

Comment:
should we do this to be consistent with the `x-chain-blocks` example?
```suggestion
		uri              = fmt.Sprintf("%s/ext/index/P/block", primary.LocalAPIURI)
```